spec/support/orm/mongo_mapper.rb

```diff
@@ -7,4 +7,20 @@
     Doorkeeper::AccessGrant.create_indexes
     Doorkeeper::AccessToken.create_indexes
   end
+
+  # This is a hack to fix a bug in a spec test in Doorkeeper.
```

Comment:
Already merged to master

Reply:
Now, it wasn't when I wrote it.  I'll take it out.